### PR TITLE
[chore] fix some simple typos

### PR DESCRIPTION
**Pull request by c8ef**

### CODE REVIEW

**Comment by vladimir-paramuzov** [comment]
build_jenkins

**Review by vladimir-paramuzov — APPROVED** [review]

File: src/plugins/auto_batch/tests/unit/plugin_compile_model_test.cpp [hunk 1]
```diff
@@ -130,7 +130,7 @@ TEST_P(PluginCompileModelTest, PluginCompileModelBatchedModelWithRemoteContextTe
 }
 
 const std::vector<plugin_compile_model_param> plugin_compile_model_param_test = {
-    // Case 1: explict apply batch size by config of AUTO_BATCH_DEVICE_CONFIG
+    // Case 1: explicit apply batch size by config of AUTO_BATCH_DEVICE_CONFIG
```
Comment (riverlijunjie) [inline src/plugins/auto_batch/tests/unit/plugin_compile_model_test.cpp:133]:
```suggestion
    // Case 1: explicitly apply batch size by config of AUTO_BATCH_DEVICE_CONFIG
```

File: src/plugins/auto/src/compiled_model.cpp [hunk 2]
```diff
@@ -63,7 +63,7 @@ ov::AnyMap ov::auto_plugin::CompiledModel::get_device_supported_properties(AutoC
     OPENVINO_ASSERT(context.m_compiled_model);
     auto device_supported_properties = context.m_compiled_model->get_property(ov::supported_properties.name());
     for (auto&& property_name : device_supported_properties.as<std::vector<ov::PropertyName>>()) {
-        // for lto issue, explictly do the conversion here
+        // for lto issue, explicitly do the conversion here
```
Comment (riverlijunjie) [inline src/plugins/auto/src/compiled_model.cpp:66]:
```suggestion
        // For LTO issue, explicitly do the conversion here
```

**Review by riverlijunjie — APPROVED** [review]

**Comment by c8ef** [comment]
I'm not entirely certain, but the CI failure appears unrelated to my changes. Please let me know if I can assist in moving this forward!

**Comment by c8ef** [comment]
I'm quite puzzled as to why the ONNX Models Tests keep failing. I haven't even modified that code.

**Comment by vladimir-paramuzov** [comment]
@c8ef Sometimes jobs fail due to connection issue or some other sporadic issues. Please don't push each time you see the failure, we can just restart failed jobs manually if needed.

**Comment by c8ef** [comment]
Thank you for your guidance! My motivation is that perhaps some flaky test cases will be resolved in the latest main branch. Therefore, I believe updating the branch would be beneficial for the CI.

**Review by ilya-lavrenov — APPROVED** [review]

**Comment by ilya-lavrenov** [comment]
build_jenkins

**Comment by c8ef** [comment]
Great, it seems that adding `pytest-randomly` has really improved the CI! The only failing CI test now is `ie_tests_cldnn_unit_ubuntu20_release`, which already passed in commit 72ea9f3. I suspect it may be flaky. Can you assist me in retriggering the failed CI test?

**Pull request merged**